addons/rating/static/src/models/message/message.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+import { registerFieldPatchModel, registerClassPatchModel } from '@mail/model/model_core';
+import { attr } from '@mail/model/model_field';
+
+registerFieldPatchModel('mail.message', 'rating/static/src/models/message.message.js', {
+    ratingImg: attr(),
```

Comment:
need to add docstring (and as per suggestion, we probably want to have also the rating value itself, and not just the img)

Also this name should reflect that it's the image URL (and not the image content itself)